libs/platform/src/lib/components/form/text-area/text-area.component.html

```diff
@@ -2,7 +2,6 @@
     #textareaElement
     fd-form-control
     [compact]="contentDensity == 'compact'"
```

Comment:
Let's follow the same performance approach we did for other components, have isCompact internal property and set it / cache whatever you need to do so we dont have comparission in here. You can check out stepinput, panel I think

Reply:
@fkolar done, please review again